### PR TITLE
feat: adding functions feature flag @W-7958729@

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Adds a feature flag for functions support

### What issues does this PR fix or reference?
@W-7958729@

### Functionality Before
N/A

### Functionality After
Screen Shot 2020-09-24 at 9 38 27 AM

### CODE REVIEW

**Comment by jag-sfdc** [comment]
These changes were already reviewed by @ekapner and @xyc as part of another review. We were waiting for other functions changes to go in before merging these.

**Review by AnanyaJha — APPROVED** [review]

**Review by xyc — APPROVED** [review]

File: packages/salesforcedx-vscode-core/package.nls.json [hunk 1]
```diff
@@ -74,5 +74,5 @@
   "force_function_start": "SFDX: Start Function",
   "force_function_invoke": "SFDX: Invoke Function",
   "force_function_stop": "SFDX: Stop Function",
-  "experimental_templates_library_description": "Use Templates Library for Templates CLI commands"
```
Comment (xyc) [inline packages/salesforcedx-vscode-core/package.nls.json:77]: Are we removing this flag?

Comment (jag-sfdc) [reply packages/salesforcedx-vscode-core/package.nls.json:77]: This wasn't my change. I saw this defined twice. Once in line 73. Can you confirm I'm doing this right.

Comment (xyc) [reply packages/salesforcedx-vscode-core/package.nls.json:77]: Yes let's remove this. maybe it was a merge conflict

**Pull request merged**